tool/src/main/java/org/datacommons/tool/Tool.java

```diff
@@ -0,0 +1,231 @@
+package org.datacommons.tool;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.datacommons.proto.Debug;
+import org.datacommons.proto.Mcf;
+import org.datacommons.util.McfParser;
+import org.datacommons.util.McfUtil;
+import org.datacommons.util.TmcfCsvParser;
+import picocli.CommandLine;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.Callable;
+
+// TODO: Add e2e tests once Debug.Log is fully plumbed in.
+@CommandLine.Command(
+    name = "dc-import",
+    mixinStandardHelpOptions = true,
+    version = "dc-import 0.1",
+    description = "Tool for use in developing datasets for Data Commons.",
+    subcommands = {Lint.class, GenMcf.class})
+class Tool {
+  @CommandLine.Option(
+      names = {"-o", "--outputDir"},
+      description = "Directory to write output files. Default is current working directory.",
+      scope = CommandLine.ScopeType.INHERIT)
+  public File outputDir;
+
+  public static void main(String... args) {
+    System.exit(new CommandLine(new Tool()).execute(args));
+  }
+
+  public static void HandleNodeFormat(
+      Mcf.McfType type, File file, Debug.Log.Builder logCtx, Logger logger) throws IOException {
+    int numNodesProcessed = 0;
+    logger.debug("Checking {}", file.getName());
+    // TODO: isResolved is more allowing, be stricter.
+    McfParser parser = McfParser.init(type, file.getPath(), false);
+    Mcf.McfGraph n;
+    while ((n = parser.parseNextNode()) != null) {
+      numNodesProcessed++;
+    }
+    logger.info("Checked {} with {} nodes", file.getName(), numNodesProcessed);
+  }
+
+  public static void HandleTableFormat(
+      File tmcfFile,
+      List<File> csvFiles,
+      char delimiter,
+      BufferedWriter writer,
+      Debug.Log.Builder logCtx,
+      Logger logger)
+      throws IOException {
+    logger.debug("TMCF " + tmcfFile.getName());
+    for (File csvFile : csvFiles) {
+      logger.debug("Checking CSV " + csvFile.getPath());
+      TmcfCsvParser parser =
+          TmcfCsvParser.init(tmcfFile.getPath(), csvFile.getPath(), delimiter, logCtx);
+      Mcf.McfGraph g;
+      int numNodesProcessed = 0, numRowsProcessed = 0;
+      while ((g = parser.parseNextRow()) != null) {
+        numNodesProcessed += g.getNodesCount();
+        numRowsProcessed++;
+        if (writer != null) {
+          writer.write(McfUtil.serializeMcfGraph(g, false));
+        }
+      }
+      logger.info(
+          "Checked CSV {} ({} rows, {} nodes)",
+          csvFile.getName(),
+          numRowsProcessed,
+          numNodesProcessed);
+    }
+  }
+}
+
+@CommandLine.Command(name = "lint", description = "Run various checks on input MCF/TMCF/CSV files")
```

Comment:
Dose lint tool output files? The command help message indicate but it's not relevant?

Reply:
Upcoming PRs will change it to output a report.